gitless/cli/gl_branch.py

```diff
@@ -55,6 +55,11 @@ def parser(subparsers, _):
       '-uu', '--unset-upstream',
       help='unset the upstream branch of the current branch',
       action='store_true')
+  edit_group.add_argument(
+      '-rn', '--rename-branch', nargs='+',
+      help='renames this branch or another specified branch',
```

Comment:
You need to create a different command group. `edit_group` has the help message `"edit the current branch"`. Perhaps rename `edit_group` to `edit_current_branch_group` and use `edit_group` with the help message `"edit branches"`?

If you do `gl branch -h` you'll notice why it matters. The option to rename would appear under "edit the current branch"